amr-wind/core/SimTime.cpp

```diff
@@ -92,7 +92,8 @@ void SimTime::set_current_cfl(
     amrex::Real dt_new = 2.0 * m_max_cfl / cfl_unit_time;
 
     // Restrict timestep during initialization phase
-    if (m_is_init) dt_new *= m_init_shrink;
+    if (m_is_init)
+        dt_new = amrex::min(m_init_shrink * dt_new, std::abs(m_fixed_dt));
```

Comment:
Remove this as this was a hack to allow you to run the dam break problem. A better fix has been added to the main branch.